pyroengine/engine.py

```diff
@@ -27,6 +27,33 @@
 logging.basicConfig(format="%(asctime)s | %(levelname)s: %(message)s", level=logging.INFO, force=True)
 
 
+def is_day_time(cache):
+    """Read sunset and sunrise hour in sunset_sunrise.txt and check if we are currently on daytime. We don't want to
+    trigger night alerts for now. We take 1 hour margin
+
+    Args:
+        cache (Path): cache folder where sunset_sunrise.txt is located
+
+    Returns:
+        bool: is day time
+    """
+    with open(cache.joinpath("sunset_sunrise.txt")) as f:
+        lines = f.readlines()
+    sunrise = lines[0][:-1].split(":")
+    sunset = lines[1][:-1].split(":")
+    th = datetime.now().hour
+    tm = datetime.now().minute
+    if (
+        th < int(sunrise[0]) - 1
+        or (th == int(sunrise[0]) - 1 and tm < sunrise[1])
+        or th > int(sunset[0]) + 1
+        or (th == int(sunset[0]) + 1 and tm > int(sunset[1]))
+    ):
```

Comment:
I suggest parsing the string with `datetime.strptime(date_string, format)` to get datetime comparison directly :)

Reply:
done :)